meshroom/ui/qml/Application.qml

```diff
@@ -1105,22 +1105,45 @@ Page {
                     for (var i = 0; i < node.attributes.count; i++) {
                         var attr = node.attributes.at(i)
                         if (attr.isOutput && attr.desc.semantic !== "image")
-                            if (!alreadyDisplay || attr.desc.semantic == "3D")
+                            if (!alreadyDisplay || attr.desc.semantic == "3D") {
```

Comment:
Not introduced by this PR but since the line has been modified... isn't there a typo here? Shouldn't the check on the semantic be for `"3d"` instead of `"3D"`? (or maybe I'm missing something else)
```suggestion
                            if (!alreadyDisplay || attr.desc.semantic == "3d") {
```